gpflow/optimizers/natgrad.py

```diff
@@ -105,11 +109,9 @@ def minimize(self, loss_fn: Callable, var_list: List[Parameter]):
         self._natgrad_steps(loss_fn, parameters)
 
     def _natgrad_steps(self, loss_fn: Callable, parameters: List[Tuple[Parameter, Parameter, XiTransform]]):
-        def natural_gradient_step(q_mu, q_sqrt, xi_transform):
-            self._natgrad_step(loss_fn, q_mu, q_sqrt, xi_transform)
-
         with tf.name_scope(f"{self._name}/natural_gradient_steps"):
-            list(map(natural_gradient_step, *zip(*parameters)))
+            for q_mu, q_sqrt, xi_transform in parameters:
+                self._natgrad_step(loss_fn, q_mu, q_sqrt, xi_transform)
```

Comment:
@vdutor in trying to understand the natgrad code, I realised this wrapper can be written much more simply.

Reply:
and another small change

Reply:
This is indeed much nicer! Have you checked `_natgrad_steps` isn't used anywhere else?

Reply:
yup